scribejava-apis/src/test/java/com/github/scribejava/apis/examples/OdnoklassnikiExample.java

```diff
@@ -18,14 +18,14 @@
 
     public static void main(String... args) {
         // Replace these with your client id and secret
-        final String clientId = "your client id";
-        final String publicKey = "your api secret";
-        final String clientSecret = "your client secret";
+        final String clientId  = "your api client id";
+        final String publicKey = "your api public key";
+        final String secretKey = "your api secret key";
 
         final OAuth20Service service = new ServiceBuilder()
                 .apiKey(clientId)
-                .apiSecret(clientSecret)
-                .scope("VALUABLE ACCESS")
+                .apiSecret(secretKey)
+                .scope("VALUABLE_ACCESS")
```

Comment:
May be better remove at all, because
"VALUABLE_ACCESS - access to all API methods, except users.getLoggedInUser and users.getCurrentUser. To have effect permission must also be set by Odnoklassniki side as well. To request permission send an email with id and shortname of your application to the address api-support@odnoklassniki.ru"